.erb-lint.yml

```diff
@@ -57,6 +57,9 @@ linters:
           - overflow-(hidden|scroll|auto)
           - (left|right)
           - border-box
+          - (bold|regular|italic|caps)
+          - '(left|right)-align'
+          - (center|justify|nowrap)
```

Comment:
Can we consolidate overlapping patterns on line 22-24?

Reply:
Yup! 39d66bed0